### DISCUSSION TITLE
Chamber or Trap

### DISCUSSION THREAD

**Discussion by tomakor549** [HaeffnerLab/IonSim.jl#116] [Q&A] [answered]
I work on a project that uses IonSim Trap structure, and I'm wondering, what's the difference between Trap and Chamber? Chamber seems to be a newer, more developed version of Trap. Would it be worth to update the code so that it uses Chamber instead?

**Comment by neil-glikin** [comment, marked as answer]
Yes, v0.5.0 included some renames, one of which was renaming `Trap` to `Chamber`. If you want to update from a version before 0.5.0 to one after, you'll need to change your code to use `Chamber` instead of `Trap`.
